extensions/amp-analytics/0.1/visibility-manager.js

```diff
@@ -363,6 +363,7 @@ export class VisibilityManager {
 
       } else {
         state['opacity'] = this.getRootMinOpacity();
+        state['intersectionRatio'] = this.getRootVisibility();
```

Comment:
One question I have is that `interesectionRatio` doesn't make much sense for root other than inabox use case. And if we also need to update the document?

Reply:
I agree `intersectionRatio` doesn't make much sense for `root` outside the Inabox case, but I think it's not a problem to support it?

By "And if we also need to update the document?" did you mean the documentation?  I just added that: 9f286fc3a

Reply:
Haha totally meant documentation! LTGM, Thanks!